### ISSUE TITLE
PHP notice/warning database conversion

### ISSUE THREAD

**Issue by Vertux** [friendica/friendica#7128]
Since a while I get these PHP messages 

I am not sure, if it is something I should worry about and if there is a way to fix it.

Version: Friendica 'Dalmatian Bellflower' 2019.04 - 1304 
PHP: PHP Version 7.2.18
MySQL/MariaDB: 10.1.38 

```
PHP Notice:  Array to string conversion in /friendica/src/Model/APContact.php on line 159
PHP Warning:  trim() expects parameter 1 to be string, array given in /friendica/src/Model/APContact.php on line 161
PHP Notice:  Array to string conversion in /friendica/src/Database/DBA.php on line 485
PHP Notice:  Array to string conversion in /friendica/src/Database/DBA.php on line 485
PHP Notice:  Array to string conversion in /friendica/src/Database/DBA.php on line 485
PHP Notice:  Array to string conversion in /friendica/src/Database/DBA.php on line 485
PHP Notice:  Array to string conversion in /friendica/src/Model/GContact.php on line 825
PHP Notice:  Array to string conversion in /friendica/src/Model/GContact.php on line 825
PHP Notice:  Array to string conversion in /friendica/src/Database/DBA.php on line 485
PHP Notice:  Array to string conversion in /friendica/src/Database/DBA.php on line 485
```

**Comment by annando** [comment]
This is some ActivityPub issue that the stable version has got with Pleroma. It is already solved with the develop version.

**Comment by MrPetovan** [comment]
Fixed with #7045
